src/Hadrian/Haskell/Cabal/Parse.hs

```diff
@@ -109,6 +109,9 @@ configurePackage :: Context -> Action ()
 configurePackage context@Context {..} = do
     putLoud $ "| Configure package " ++ quote (pkgName package)
 
+    root <- buildRoot
+    need [root -/- stageString stage -/- "lib/package.conf.d" -/- packageDbStamp]
```

Comment:
I think we've got some helper function to get that path given the stage, `relativePackageDbPath` or something like that.

Reply:
We can't use `relativePackageDbPath` here, because it does `min stage Stage1` on the given `stage`, but we really want the path to the Stage2 package database here. 

I hope we'll be able to find a more elegant way of dealing with package databases.

Reply:
Should it still do the `min` operation?

Reply:
I think yes, because we need to point Stage2 GHC to the right package database, which lives in `_build/stage1/lib`.

Unless you suggest to build all libraries again, using Stage2 GHC and build a new package database in `_build/stage2/lib`.